core/src/main/java/bisq/core/account/witness/AccountAgeWitnessService.java

```diff
@@ -425,7 +426,7 @@ private long getTradeLimit(Coin maxTradeLimit,
                                PaymentMethod paymentMethod) {
         if (CurrencyUtil.isCryptoCurrency(currencyCode) ||
                 !PaymentMethod.hasChargebackRisk(paymentMethod, currencyCode) ||
-                direction == OfferPayload.Direction.SELL) {
+                direction == OfferPayloadI.Direction.SELL) {
```

Comment:
The naming of the interface `OfferPayloadI` somehow feels weird. How about calling the interface `OfferPayload` and the implementation `OfferPayloadImpl` and coding as much as possible against the interface? Or call the implementation something other more specific (I'm to early in the commits to suggest something better - `OfferWithFeePayload`?)

Reply:
Or is this about the proto resolving to be backwards compatible?